projects/pastanaga/src/lib/controls/checkbox-tree/checkbox-tree.component.ts

```diff
@@ -215,7 +215,7 @@ export class CheckboxTreeComponent implements ControlValueAccessor, OnInit, OnCh
 
     private sortCheckboxes(checkboxes: ControlModel[]): ControlModel[] {
         if (checkboxes && checkboxes.length > 0) {
-            this.translate.stream(checkboxes.map(checkbox => checkbox.label || '')).subscribe((labels => {
+            checkboxes.map(checkbox => this.translate.transform(checkbox.label || '')).map((labels => {
```

Comment:
`map` is returning an array done based on the array given in entry. Here what you want is `forEach`